src/documentation/config/config.md

```diff
@@ -38,6 +38,12 @@ Each configuration option can be:
 
 		columnWidth: function() { return this.rangeBand/3 * 2; }
 
+	__Note__ Function parameters will be invoked just before each render happens or in the within the render cycle with the corresponding data point in the following cases:
+	* function expects a parameter (ie. `function (d) { return d.x; }`)
+	* parameter path is in the `options.skip`
+	* parameter path matches the RegExp `options.skipMatch`
```

Comment:
Since you're computing the "path" anyway, in which cases would just the skip array not be good enough?

Reply:
I change the skip array to accept full paths, or regexp so you have a simple list of things to skip. I think the regexp is handy to skip global params like formatter (ie `/formatter/`) without having to specify every path that you want to skip.